Jenkinsfile

```diff
@@ -52,7 +52,7 @@ def configureDocker = {
 
 def installQpcClient() {
     sh '''\
-    sudo wget -O /etc/yum.repos.d/chambridge-qpc-fedora-28.repo https://copr.fedorainfracloud.org/coprs/chambridge/qpc/repo/fedora-28/chambridge-qpc-fedora-28.repo
+    sudo wget -O /etc/yum.repos.d/group_quipucords-qpc-fedora-29.repo https://copr.fedorainfracloud.org/coprs/g/quipucords/qpc/repo/fedora-28/group_quipucords-qpc-fedora28.repo
```

Comment:
We are using Fedora 28 to build this, so the repo should be the one for that OS.

Reply:
Got it. I was unsure about which to use.

Reply:
actually looking at this... it is using 28, but pulling it down to a file named 29. I'll fix that...